src/python/gudhi/perslay.py

```diff
@@ -0,0 +1,263 @@
+# This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+# See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+# Author(s):       Mathieu Carriere
+#
+# Copyright (C) 2019 Inria
+#
+# Modification(s):
+#   - YYYY/MM Author: Description of the modification
+
+import numpy as np
+import tensorflow as tf
+
+
+# Post-processing operation with combination of batch normalization, dropout and relu
+def _post_processing(vector, pro, dropout_value=.9):
+    for c in pro:
+        if c == "b":
+            vector = tf.layers.batch_normalization(vector)
+        if c == "d":
+            vector = tf.nn.dropout(vector, dropout_value)
+        if c == "r":
+            vector = tf.nn.relu(vector)
+    return vector
+
+# Vectorization implementing DeepSet architecture
+def permutation_equivariant_layer(inp, dimension, perm_op, L_init, G_init, bias_init, L_const, G_const, bias_const, train_vect):
+    """ DeepSet PersLay """
+    dimension_before, num_pts = inp.shape[2].value, inp.shape[1].value
+    lbda = tf.get_variable("L", shape=[dimension_before, dimension], initializer=L_init, trainable=train_vect)   if not L_const     else tf.get_variable("L", initializer=L_init)
+    b    = tf.get_variable("b", shape=[1, 1, dimension], initializer=bias_init, trainable=train_vect)            if not bias_const  else tf.get_variable("b", initializer=tf.reshape(bias_init, [1, 1, dimension]))
```

Comment:
You have a number of very long lines like that. While the PEP8 limit of 79 characters per line seems too small to me, 211 on this line is too much. Could you consider ways to reduce super-long lines? For instance, this one could easily be split, although it loses the alignment with `A = ...` below. Running `black -l N perslay.py` (where N might be 79, 120, whatever line length) is another way to avoid long lines.
```suggestion
    if bias_const:
        b = tf.get_variable("b", initializer=tf.reshape(bias_init, [1, 1, dimension]))
    else:
        b = tf.get_variable("b", shape=[1, 1, dimension], initializer=bias_init, trainable=train_vect)
```